jetty-deploy/src/test/java/org/eclipse/jetty/deploy/providers/WebAppProviderTest.java

```diff
@@ -159,14 +157,25 @@ public void testWebappSymlinkDir() throws Exception
         Files.createDirectory(x.toPath());
 
         //Put a webapp into it
-        File srcDir = MavenTestingUtils.getTestResourceDir("webapps");
+        Path srcDir = MavenTestingUtils.getTestResourcePathDir("webapps");
```

Comment:
None of the changes to this test are relevant to the change to scanning. Are you sure these are vital?

Reply:
Changed to using non-deprecated test functions.